cime_config/machines/config_batch.xml

```diff
@@ -514,6 +514,16 @@
 	</queues>
     </batch_system>
 
+   <batch_system MACH="chicoma-cpu" type="slurm">
+    <directives>
+     <directive>--partition=standard </directive>
```

Comment:
Current project only has access to gpu partition so --partition=gpu needed here unless there is another project besides t22_e3sm-gpu_g

Reply:
We will add `chicoma-gpu` later but the idea here was to start with CPU (more familiar to most of us).  We have several projects I believe with Chicoma CPU time (e.g. `t22_ocean_time_step`).

Reply:
ok, we should keep this then as long as we change the project name